Router/Router/App/LoginScreen/View/LoginView.swift

```diff
@@ -0,0 +1,107 @@
+    private func setupView() {
+        backgroundColor = .white
+    }
+    private func setupLoginTextField() {
+        loginTextField.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(loginTextField)
+        NSLayoutConstraint.activate([
+            loginTextField.topAnchor.constraint(equalTo: topAnchor, constant: 200),
+            loginTextField.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 50),
+            loginTextField.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -50),
+            loginTextField.heightAnchor.constraint(equalToConstant: 30)
+        ])
+        loginTextField.placeholder = " Login"
+        loginTextField.layer.cornerRadius = 5
+        loginTextField.layer.borderColor = UIColor.gray.cgColor
+        loginTextField.layer.borderWidth = 0.5
+    }
+
+    private func setupPasswordTextField() {
+        passwordTextField.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(passwordTextField)
+        NSLayoutConstraint.activate([
+            passwordTextField.topAnchor.constraint(equalTo: loginTextField.bottomAnchor, constant: 50),
+            passwordTextField.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 50),
+            passwordTextField.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -50),
+            passwordTextField.heightAnchor.constraint(equalToConstant: 30)
+        ])
+        passwordTextField.placeholder = " Password"
+        passwordTextField.isSecureTextEntry = true
+        passwordTextField.layer.cornerRadius = 5
+        passwordTextField.layer.borderColor = UIColor.gray.cgColor
+        passwordTextField.layer.borderWidth = 0.5
+    }
+
+    private func setupLoginButton() {
+        loginButton.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(loginButton)
+        NSLayoutConstraint.activate([
+            loginButton.topAnchor.constraint(equalTo: passwordTextField.bottomAnchor, constant: 100),
+            loginButton.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -100),
+            loginButton.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 100),
+            loginButton.heightAnchor.constraint(equalToConstant: 50)
+        ])
+        loginButton.layer.cornerRadius = 5
+        loginButton.backgroundColor = .black
+        loginButton.setTitle("Log In", for: .normal)
+        loginButton.setTitleColor(.white, for: .normal)
+        loginButton.addTarget(self, action: #selector(nextVC), for: .touchUpInside)
+    }
+
+    private func configureLoginBindings() {
+        Observable
+            .combineLatest(loginTextField.rx.text, passwordTextField.rx.text)
+            .map { login, password in
+                return !(login ?? "").isEmpty && !(password ?? "").isEmpty
+            }
+            .subscribe { [weak loginButton] inputFilled in
+                loginButton?.isEnabled = inputFilled
+            }
+    }
+
+    @objc private func nextVC() {
```

Comment:
```suggestion
    @objc private func onTap() {
```
вью не знает что будет переход она тупая